Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.view.html

```diff
@@ -207,6 +207,22 @@ <h4 data-element-type="dpaMainDetailHeader">{{vm.headerName}}</h4>
                             </tr>
                         </tfoot>
                     </table>
+                    <div class="form-group">
+                        <label for="last-changed-by-name" class="control-label small">Sidst redigeret: Bruger</label>
+                        <input type="text"
+                               id="last-changed-by-name"
+                               value="{{vm.lastChangedBy}}"
+                               class="form-control input-sm"
+                               disabled>
+                    </div>
+                    <div class="form-group">
+                        <label for="last-changed" class="control-label small">Sidst redigeret: Dato</label>
+                        <input type="text"
+                               id="last-changed"
+                               value="{{vm.lastChangedAt}}"
+                               class="form-control input-sm"
+                               disabled>
+                    </div>
```

Comment:
Think you need to work more on your QA skills
![image](https://user-images.githubusercontent.com/13116889/188849334-5160e368-504b-4b4d-8b5e-04a1f95825db.png)
The screenshot is from the default DPA that comes with the clean db.
You placed the markup inside a group which is guarded by a condition